templates/workspaces/base_workspace/terraform/locals.tf

```diff
@@ -0,0 +1,13 @@
+data "azurerm_subscription" "current" {}
+
+data "azurerm_client_config" "current" {}
+
+# Random unique id
+resource "random_string" "unique_id" {
+  length      = 4
+  min_numeric = 4
+}
+
+locals {
+  tre_id                                = random_string.unique_id.result
```

Comment:
Should this be workspace_id ?

Reply:
Good point, id should probably be sequential; I'll change.